news/4.32/pde.html

```diff
@@ -76,11 +76,18 @@ <h2>Plug-in Development Environment</h2>
       </td>
   </tr>
   
-  <!-- ******************** End ofDialogs, Wizard and Views ********************** -->
+  <!-- ******************** End of Dialogs, Wizard and Views ********************** -->
   
   <!-- ******************** Editors ********************** -->
   <tr>
     <td id="editors" class="section" colspan="2"><h2>Editors</h2></td>
+    <td class="content">
```

Comment:
Please add comment with the link to the original issue fixed, like in the section above

Reply:
@iloveeclipse Sorry, I lost sight of this. Added it now